src/llmcompressor/modifiers/quantization/gptq/utils/gptq_wrapper.py

```diff
@@ -127,26 +130,28 @@ def compress(
             # mapping from column index to group index
             g_idx = (
                 torch.arange(self.columns, device=W.device, dtype=torch.int)
-                // weight_quant_args.group_size
+                // quant_args.group_size
             )
 
             if actorder == ActivationOrdering.GROUP:
                 # permute by activation order first, then update groups
                 W, self.H, perm = self._apply_activation_ordering(W, self.H)
-                self._update_quantization_parameters(weight_quant_args, W)
+                scale, zero_point = observer(W, g_idx=None)
```

Comment:
Do we always need to pass in g_idx?

Reply:
The default value is None, but I keep it explicit here to emphasize that the scales and zero points are being computed as if there was no g_idx (even though it has been permuted by the `_apply_activation_ordering` function)